### PR TITLE
add test for image name

### PR DESCRIPTION
**Pull request by cdeepali**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  

## Description

Remove '-runtime' string being added to the image name. 
Also add a test for image name verification.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: tests/build_image_test.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def test_local_conda_channel_with_absolute_path(mocker):
     '''
     Test for build_runtime_image with local conda channel with its absolute path
     '''
-    intended_image_name = build_image.REPO_NAME + ":" + build_image.IMAGE_NAME
+    intended_image_name = build_image.REPO_NAME + ":" + build_image.IMAGE_NAME + "-" + "test-conda-env"
```
Comment (npanpaliya) [inline tests/build_image_test.py:94]: Just a nit pick - Can we make this "test-conda-env" as constant variable which is declared once.

**Review by npanpaliya — APPROVED** [review]

**Comment by cdeepali** [comment]
@jayfurmanek, @bnemanich, do we want to add arch to the image name as well ?

**Comment by bnemanich** [comment]
@dllehr81 what do you think about the image name? I don't think we want to have the architecture name in it, but would like to verify.

File: tests/build_image_test.py (hunk 1 above)
Comment (jayfurmanek) [reply tests/build_image_test.py:94]: That's probably a good idea. That makes it easy to change if we need to.

Comment (cdeepali) [reply tests/build_image_test.py]: I have added a commit to address this. @npanpaliya, @jayfurmanek please have a look.

Comment (cdeepali) [reply tests/build_image_test.py]: Similar change would be needed in other tests too. I will create another PR for that.

**Review by npanpaliya — APPROVED** [review]
Looks good @cdeepali ! Thanks for fixing this all over.

**Review by bnemanich — APPROVED** [review]

**Comment by cdeepali** [comment]
Merging this one.

**Pull request merged**